python/lsst/pipe/tasks/postprocess.py

```diff
@@ -1011,7 +1106,9 @@ def run(self, handle, funcs=None, dataId=None, band=None):
 
         # This makes a multilevel column index, with band as first level
         df = pd.concat(dfDict, axis=1, names=["band", "column"])
+        name_index = df.index.name
 
+        # TODO: Remove in DM-???
```

Comment:
:+1: to removing this!  I don't think it's actually usable anymore, but would need to deprecate some configs, at least.

Reply:
48895 (linked above) is the ticket number.